### PR TITLE
test(gemini-ics-cdc): New test gemini + cdc + ics

### PR DESCRIPTION
**Pull request by aleksbykov**
Added new test to run gemini with enabled cdc and ics
Additionaly, run cdcreader used to read cdc log table

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~New configuration option are added and documented (in `sdcm/sct_config.py`)~
- [ ] ~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~
- [ ] ~All new and existing unit tests passed (CI)~
- [ ] ~I have updated the Readme/doc folder accordingly (if needed)~

### CODE REVIEW

**Comment by aleksbykov** [comment]
BYO job was passed: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/abykov/job/BYO-abykov-staging/81

**Comment by bentsi** [comment]
Unit tests failed
image

**Comment by aleksbykov** [comment]
@bentsi i think @amoskong prepare  a fix for tha in PRt: https://github.com/scylladb/scylla-cluster-tests/pull/3402

**Comment by amoskong** [comment]
> @bentsi i think @amoskong prepare a fix for tha in PRt: #3402

Yes.

File: gemini_test.py [hunk 1]
```diff
@@ -67,6 +67,38 @@ def test_load_random_with_nemesis(self):
...
+        cdc_stress = self.params.get('stress_cdclog_reader_cmd')
+        update_es = self.params.get('store_cdclog_reader_stats_in_es')
+
+        self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(),
+                                    tester_obj=self)
+
+        self.log.debug('Start gemini benchmark')
+        gemini_thread = self.run_gemini(cmd=cmd)
+        self.gemini_results["cmd"] = gemini_thread.gemini_commands
+        # waite gemini create schema
```
Comment (roydahan) [inline gemini_test.py:81]:
```suggestion
        # wait gemini create schema
```

File: gemini_test.py [hunk 2]
```diff
@@ -67,6 +67,38 @@ def test_load_random_with_nemesis(self):
...
+        time.sleep(15)
```
Comment (roydahan) [inline gemini_test.py:82]: Why not waiting for schema agreement or  something more deterministic if possible?

**Review by roydahan — APPROVED** [review]
Overall look good to me.
Does it pass staging runs?

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
@aleksbykov please also create the jobs in 2021.1 and link it here.

**Comment by roydahan** [comment]
Also, change the target branch to be master and we will backport it to 2021.1

File: gemini_test.py (hunk 1 above)
Comment (aleksbykov) [reply gemini_test.py:81]: fixed

File: gemini_test.py (hunk 2 above)
Comment (aleksbykov) [reply gemini_test.py:82]: Changed to wait schema agreement.

**Comment by aleksbykov** [comment]
@roydahan @bentsi fix according comments. I changed target branch to master and set backport to 2021.1

**Comment by aleksbykov** [comment]
JOb was created https://jenkins.scylladb.com/view/enterprise-2021.1/job/enterprise-2021.1/job/gemini-/job/gemini-3h-ics-cdc-with-nemesis-test/. Once backported, will be run

**Comment by roydahan** [comment]
@aleksbykov I can't merge till all check passes.
Please take a look.

**Comment by roydahan** [comment]
Merged and backported.
Please make sure it's in the spreadsheet of 2021.1 and run it on official branches.

**Pull request merged**